presto-main/src/main/java/com/facebook/presto/Session.java

```diff
@@ -269,7 +270,9 @@ public Session beginTransactionId(TransactionId transactionId, TransactionManage
                 continue;
             }
             ConnectorId connectorId = transactionManager.getOptionalCatalogMetadata(transactionId, catalogName)
-                    .orElseThrow(() -> new PrestoException(NOT_FOUND, "Catalog does not exist: " + catalogName))
+                    .orElseThrow(() ->
```

Comment:
Let's change the error message to this:
```java
.orElseThrow(() -> new PrestoException(NOT_FOUND, "Session property catalog does not exist: " + catalogName))
```
The full name of the session property is not relevant.

Reply:
The information can help to identify which property was added and caused problem. But I think knowing session property was set incorrectly should already be enough.  I will update the PR.